AdventOfCode23/Day4/CardCollection.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace AdventOfCode23.Day4;
+
+public static class CardCollection
+{
+    /// <summary>
+    ///     Calculates the total score (as described in Day 4 Problem 1) of a dataset of Cards contained in a text file at the specified path.
+    /// </summary>
+    /// <param name="path"></param>
+    /// <returns>The total score.</returns>
+    public static int CalculateTotalScore(string path)
+    {
+        return File.ReadAllLines(path).Sum(Card.CalculateScore);
```

Comment:
Yeah, so this clearly isn't the best way to traverse the dataset and perform the algorithm. However, I'm just going to ignore that for now. It's pretty simple to do a little stream reading exercise, accumulating the sum as you read through the dataset (rather than putting the whole thing into memory). However, the meat of the problem was really the `Card.CalculateScore` function, so I don't really care too much about the file reading method.